pkg/asset/installconfig/azure/validation.go

```diff
@@ -119,6 +120,37 @@ func validateRegion(client API, fieldPath *field.Path, p *aztypes.Platform) fiel
 	return field.ErrorList{field.Invalid(fieldPath, p.Region, fmt.Sprintf("region %q does not support resource creation", p.Region))}
 }
 
+// validateRegionForUltraDisks checks that the Ultra SSD disks are available for the user.
+func validateRegionForUltraDisks(fldPath *field.Path, region string) field.ErrorList {
+	allErrs := field.ErrorList{}
```

Comment:
this is a single field validation function correct? so there is going to be only one error possible. so no need to use ErrorList

simply return `field.InternalError` or `field.Invalid` ..